README.md

```diff
@@ -98,7 +98,7 @@ conda activate venv_sct
 Copy the file `configuration_default.json` and rename it as `configuration.json`. Edit it and modify according to your setup:
 
 - `path_data`: Absolute path to the input [BIDS dataset](#dataset-structure); the path should end with `/`.
-- `include-list`: List of subjects to include in the preprocessing, separated with a space.
+- `include_list`: List of subjects to include in the preprocessing, separated with a space.
```

Comment:
Thank you for the feedback @joshuacwnewton, I'll keep this in mind!

Reply:
About mentioning all the arguments under the `script_args` filed in the `configuration.json`, I think this would bring in the possibility of confusion (they might just overlook that the `script_args` have to be changed and might just change the values of `x`) and errors from the users side, especially when the user would have to figure out the values of the fields from the field `x` in the `configuration.json` file on their own. IMO, the current structure of the `configuration.json` is pretty self-explanatory. What do you think?

Reply:
About mentioning all the arguments under the `script_args` filed in the `configuration.json`, I think this would bring in the possibility of confusion (they might just overlook that the `script_args` have to be changed and might just change the values of `x`) and errors from the users side, especially when the user would have to figure out the values of the fields from the field `x` in the `configuration.json` file on their own. IMO, the current structure of the `configuration.json` is pretty self-explanatory. What do you think?